builder/builder.go

```diff
@@ -1538,15 +1543,25 @@ func (b *Builder) BuildUpdate(prefixflag string, minVersion int, format string,
 	return nil
 }
 
-func (b *Builder) buildUpdateContent(timer *stopWatch, mixVersion uint32, minVersion uint32, format uint32, skipSigning bool) error {
+func (b *Builder) buildUpdateContent(params UpdateParameters, timer *stopWatch) error {
 	var err error
 
+	// TODO: move this to parsing configuration / parameter time.
+	// TODO: should this be uint64?
```

Comment:
yes.. but we need to update this all over the code to be s/uint32/uint64 or just uint.

Reply:
this TODO comment just moved along with the code below, it is not my comment.

Reply:
No prob :) Just reiterating so everyone knows I want to change this in the code at some point.